docker-compose.yml

```diff
@@ -21,12 +22,26 @@ services:
     command: rails jobs:work
     depends_on:
       - db
+      - redis
     environment:
       RAILS_ENV: development
       DATABASE_URL: postgres://postgres:mysecretpassword@db:5432/postgres
+      REDIS_URL: redis://redis:6379
   webpacker:
     build: .
     command: ./bin/webpack-dev-server
+  sidekiq:
+    build: .
+    command: bundle exec sidekiq -c 2
```

Comment:
Set the concurrency to a low value as we don't need much in development and it's easy to hit PG connection pool issues otherwise.